src/graphql/queries/ccps.ts

```diff
@@ -7,13 +7,28 @@ export interface CCP {
   eventId: Event;
 }
 
+export const GET_CCP_BY_ID = gql`
+  query collectionPoint($id: ID!) {
+    collectionPoint(id: $id) {
+      id
+      name
+      eventId {
+        id
+        name
+        eventDate
+      }
+    }
+  }
+`;
+
 export const GET_ALL_CCPS = gql`
   query {
     collectionPoints {
       id
       name
       eventId {
-        id
+        name
+        eventDate
```

Comment:
removing `id` might break other uses of this `GET_ALL_CCPS` query
```suggestion
        id
        name
        eventDate
```